skip-tests.json

```diff
@@ -1674,6 +1674,7 @@
                 "deposit": "not provided",
                 "withdraw": "not provided"
             }
-        }
+        },
+        "skipWs": true
```

Comment:
@sc0Vu why are we skipping it?

Reply:
@carlosmiei because of the currency not found

Reply:
@sc0Vu We can make granular skips instead of skipping everything